packages/vaadin-combo-box/src/vaadin-combo-box-dropdown.js

```diff
@@ -285,7 +286,15 @@ class ComboBoxDropdown extends mixinBehaviors(IronResizableBehavior, PolymerElem
     return !this.loading && !(this._items && this._items.length);
   }
 
+  _positionTargetChanged() {
+    // we must update the overlay width when the positionTarget is set (or changes)
+    this._setOverlayWidth();
```

Comment:
Let's check if target is set to avoid running this when observer runs for default `undefined` value.

```suggestion
  _positionTargetChanged(target) {
    // we must update the overlay width when the positionTarget is set (or changes)
    if (target) {
      this._setOverlayWidth();
    }
```

Reply:
Done.